server/flaskr/database/postgres/handlers/decision_data_handler.py

```
@@ -0,0 +1,75 @@
+                    description,
+                    status,
+                    initialDate,
+                    dueDate,
+                    None,
+                    responsibleId,
+                    meetingId,
+                )
+
+                cur.execute(query, params)
+
+                decisionId, decisionNb = cur.fetchone()
```

Ruff (0.14.14)

[warning] Unpacked variable `decisionNb` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)